cmd/posix-list-dir-others.go

```diff
@@ -108,3 +108,44 @@ func readDirN(dirPath string, count int) (entries []string, err error) {
 	}
 	return entries, nil
 }
+
+// getDiskUsage returns du information for the posix path, recursively.
+func getDiskUsage(diskPath string, buf []byte) (uint64, error) {
+	d, err := os.Open(diskPath)
+	if err != nil {
+		return 0, err
+	}
+	defer d.Close()
+
+	maxEntries := 1000
```

Comment:
Does this mean that we will support a maximum of 1000 buckets?

Reply:
No that is just for per Readdir() listing @kannappanr , the readdir proceeds to io.EOF